src/views/Project/Details/General/StakeholdersModal/SearchOrganization/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import ListView from '#rscv/List/ListView';
+import SearchInput from '#rsci/SearchInput';
+import OrganizationItem from '#components/other/OrganizationItem';
+import AddOrganizationModal from '#components/other/AddOrganizationModal';
+import PrimaryButton from '#rsca/Button/PrimaryButton';
+import Modalize from '#rscg/Modalize';
+
+import _ts from '#ts';
+
+import styles from './styles.scss';
+
+const PrimaryModalButton = Modalize(PrimaryButton);
+
+interface Props {
+    className?: string;
+    organizationList: Organization[];
+    handleSearch: (searchText?: string) => void;
+}
+
+interface Organization {
+    id: number;
+    title: string;
+    shortName: string;
+    longName: string;
+    url: string;
+}
+
+const organizationKeySelector = (d: Organization) => d.id;
+// eslint-disable-next-line
+
+function SearchOrganization(props: Props) {
+    const {
+        className,
+        organizationList,
+        handleSearch,
+    } = props;
+
+    const [searchText, setSearchValue] = useState<string|undefined>(undefined);
+
+    useEffect(() => {
+        const timeoutId = setTimeout(() => {}, 1000);
+        return () => clearTimeout(timeoutId);
+    }, [searchText]);
+
+    const handleChange = useCallback((value?: string) => {
+        setSearchValue(value);
+        if (handleSearch) {
+            handleSearch(value);
+        }
+    }, [setSearchValue, handleSearch]);
+
+    const renderEmpty = () => (
+        <div className={styles.emptyComponent}>
+            { organizationList.length === 0 && !searchText ? (
+                _ts('assessment.metadata.stakeholder', 'typeToSearchOrganizationMessage')
+            ) : (
+                _ts('assessment.metadata.stakeholder', 'noResultsFoundMessage')
+            )}
+        </div>
+    );
```

Comment:
This should be outside the component.
We should have two Message components and switch between them